packages/util/src/generator/index.ts

```diff
@@ -0,0 +1 @@
+export * from './unique-name-generator';
```

Comment:
I think there is no need to move it into a separate directory anymore as there will be a different package for generators.